api/src/iot/import_utils.py

```diff
@@ -491,23 +512,30 @@ def parse_bulk_xlsx(workbook: Workbook) -> Generator[SensorData, None, None]:
     )
 
 
-def get_location(sensor_data: SensorData):
+def get_location(sensor_data: SensorData) -> Dict:
     """
     Get the specific django field and value which should be filled for the
-    location data that was provided.
+    location data that was provided. It will return a dict of one or multiple locations.
     """
-    if isinstance(sensor_data.location, Regions):
-        return {'regions': sensor_data.location.regions}
-    elif isinstance(sensor_data.location, LatLong):
-        return {'location': Point(sensor_data.location.longitude, sensor_data.location.latitude)}
-    elif isinstance(sensor_data.location, LocationDescription):
-        return {'location_description': sensor_data.location.description}
-    elif isinstance(sensor_data.location, PostcodeHouseNumber):
+    locations = {}  # empty dict to hold the locations.
+    if isinstance(sensor_data.location.region, Regions):
```

Comment:
Ik vind deze `isinstance` verwarrend. Als het geen `Regions` type is, dan is het `NoneType`, oftewel:

`if sensor_data.location.regions is not None`

Maar ik denk dat het toch beter is om een lege string te gebruiken voor "geen gebieden" (zoals ik hierboven heb laten zien). Daardoor wordt dit gewoon een truthy check...

`if sensor_data.location.regions`